client/lib/sort_preference.dart

```diff
@@ -26,7 +26,7 @@ class SortMode extends ChangeNotifier {
 
   // This type should not be a widget(e.g. Icon) because of mockito support...
   IconData get icon =>
-      _isDescOrder ? Icons.vertical_align_top : Icons.vertical_align_bottom;
+      _isDescOrder ? Icons.vertical_align_bottom : Icons.vertical_align_top;
```

Comment:
아이콘마다 이미 정해진 시멘틱이 있어서 이건 좀 아쉽네요.

그냥 제너럴 애로우가 나을 것 같습니다.

그리고 아이콘은 텍스트와 일치해야 됩니다.

현재 상태가 desc order 면 asc order로 전환하는 텍스트를 보여줘야 되고 asc order 는 위 화살표 시맨틱을 사용합니다.

```suggestion
      _isDescOrder ? Icons.arrow_upward : Icons.arrow_downward;
```